app/views/to_do_lists/_list.html.erb

```diff
@@ -3,10 +3,10 @@
   <div><%= list.description %></div>
 </div>
 
-<ul class="list-items">
-  <% list.items.each do |item| %>
-    <%= render partial: 'to_do_items/item', locals: {item: item} %>
-  <% end %>
-</ul>
+<%= render partial: 'incomplete_list', locals: {items: list.items.select {|item| item.done == false}} %>
 
-<%= link_to "#{t 'item.add.link'}", new_to_do_item_path(id: list.id), id: "add-item-link-#{list.id}" %>
\ No newline at end of file
+<%= render partial: 'to_do_items/add_alert', locals: {id: list.id} %>
+
+<%= link_to "#{t 'item.add.link'}", new_to_do_item_path(id: list.id), id: "add-item-link-#{list.id}" %>
+
+<%= render partial: 'complete_list', locals: {items: list.items.select {|item| item.done == true}} %>
```

Comment:
Noooooo! :) Remember that any logic in a view is banned. I suggest to prepare all needed data on controller.